plugin/pkg/shoot/oidc/clusteropenidconnectpreset/admission_test.go

```diff
@@ -35,7 +35,7 @@ var _ = Describe("Cluster OpenIDConfig Preset", func() {
 
 		BeforeEach(func() {
 			namespace := "my-namespace"
-			shootName := "shoot"
+			shootName := "shoot0"
```

Comment:
No need to change the name
```suggestion
			shootName := "shoot"
```